routes/Students.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router()
+
+
+const studentInfo = [
+  { id: 1, name: 'Bademus James', age: '23', grade: 'B', contact: 'Adekunle Ajasin' },
+  { id: 2, name: 'Cross Base', age: '25', grade: 'A', contact: '345 fedreckAbel street' },
+  { id: 3, name: 'Zacs Berdex', age: '29', grade: 'D', contact: 'Wanner Bros inert' },
+  { id: 4, name: 'Maset Such', age: '20', grade: 'C', contact: '34, dexter road, close to' }
+];
+
+router.get('/:id', (req, res) => {
+    let student = studentInfo.find(s => s.id === parseInt(req.params.id));
+  
+    if (student) {
+      res.send(student);
+    } else {
+      res.status(404).send('Student not found');
+    }
+  });
+  
+  router.get('/', (req, res) => {
+    res.send(studentInfo);
+  });
+  
+  router.post('/', (req, res) => {
+    let student = {
+      id: studentInfo.length + 1,
```

Comment:
Well done. You don't need to hard cord the id. 
I suggest you use mongoose to create your data schema :+1: